addons/tls13-middlebox-ghack/bootstrap.js

```diff
@@ -0,0 +1,290 @@
+"use strict";
+
+let {classes: Cc, interfaces: Ci, utils: Cu, results: Cr} = Components;
+
+Cu.import("resource://gre/modules/XPCOMUtils.jsm");
+
+XPCOMUtils.defineLazyModuleGetter(this, "Preferences", "resource://gre/modules/Preferences.jsm");
+XPCOMUtils.defineLazyModuleGetter(this, "TelemetryController", "resource://gre/modules/TelemetryController.jsm");
+
+XPCOMUtils.defineLazyServiceGetter(this, "certDB", "@mozilla.org/security/x509certdb;1", "nsIX509CertDB");
+XPCOMUtils.defineLazyServiceGetter(this, "uuidGenerator", "@mozilla.org/uuid-generator;1", "nsIUUIDGenerator");
+
+const VERSION_MAX_PREF = "security.tls.version.max";
+const VERSION_FALLBACK_LIMIT_PREF = "security.tls.version.fallback-limit";
+
+const CERT_USAGE_SSL_CLIENT      = 0x0001;
```

Comment:
Not all of these seem to be used.

Reply:
What is the best reference for these values? http://searchfox.org/mozilla-central/source/security/nss/lib/certdb/certt.h#455-467 ?